roles/adminer/README.rst

```diff
@@ -0,0 +1,72 @@
+Ansible role to install and configure adminer with docker-compose.
+Adminer is previously known as phpmyadmin.
```

Comment:
Nope. Adminer is an alternative to phpmyadmin. I would not mention it here at all.